tests/operators/test_python_operator.py

```diff
@@ -163,7 +170,8 @@ def test_python_callable_arguments_are_templatized(self):
             recorded_calls[0],
             Call(4,
                  date(2019, 1, 1),
-                 "dag {} ran on {}.".format(self.dag.dag_id, DEFAULT_DATE.date().isoformat()))
+                 "dag {} ran on {}.".format(self.dag.dag_id, DEFAULT_DATE.date().isoformat()),
```

Comment:
I think we should add a test when the named tuple's  value will contain jinja's  template '{{}}', For now we just check that the named tuple is preserved, not that the template resolution works in the tuple. 

Also I think it would be great to separate test that only tests template rendering. Maybe that's the right time. I don't think we have such test ?